tests/test_cve_api.py

```diff
@@ -0,0 +1,36 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from cvelib.cve_api import CveApi, CveRecord, CveRecordValidationError
+
+
+def test_full_schema_validation():
+    with open(Path(__file__).parent / "data/CVEv5_advanced-example.json") as record_file:
+        cve_json = json.load(record_file)
+    try:
+        CveRecord.validate(cve_json, CveRecord.Schemas.V5_SCHEMA)
+    except CveRecordValidationError as exc:
+        pytest.fail(f"{exc}: {exc.errors}")
+
+
+def test_container_schema_validation():
+    with open(Path(__file__).parent / "data/CVEv5_basic-example.json") as record_file:
+        cve_json = json.load(record_file)
+
+    cve_container = CveApi._extract_cna_container(cve_json)
+
+    try:
+        CveRecord.validate(cve_container, CveRecord.Schemas.CNA_PUBLISHED)
+    except CveRecordValidationError as exc:
+        pytest.fail(f"{exc}")
```

Comment:
Should this match above?
```suggestion
        pytest.fail(f"{exc}: {exc.errors}")
```

Reply:
It should! Will fix.